### PR TITLE
[3.7] Isis Flat Repaint - Minor Fixes

### PR DESCRIPTION
**Pull request by ciar4n**
Pull Request for Issue #12343 
### Summary of Changes

A PR for any issues with the 3.7 Isis flat repaint (#11832). Currently removes text-shadows and fixes the mouseover on modal save button. Any other noticed issues, feel free to mention them in the comments.
### Testing Instructions

Apply patch and check that all text has text-shadow removed

**Before Patch**
remove-shadow1

**After Patch**
remove-shadow2

 Open an existing menu item (Single Article). Click edit on the 'Select Article' field. Hover over 'Save'

**Before Patch**
modal-save1

**After Patch**
modal-save2
### Documentation Changes Required

None

### CODE REVIEW

**Comment by andrepereiradasilva** [comment]
there are some text shadows left, we can hardly notice but they are there

Check this elements css and you will notice the `text-shadow` there.
image

as for the rest. all fine

**Comment by ciar4n** [comment]
Thank you @andrepereiradasilva 

Changed to individual removal of text-shadow which resolves this.

File: administrator/templates/isis/css/template.css [hunk 1]
```diff
@@ -7632,7 +7633,7 @@ h6 {
 .sidebar-nav .nav-list > li > a {
   color: #555;
   padding: 3px 25px;
-  margin-left: -26px;
+  margin-left: -25px;
```
Comment (dgt41) [inline administrator/templates/isis/css/template.css]: should be 26px

**Comment by ciar4n** [comment]
Thank you @dgt41 . Amended.

**Comment by andrepereiradasilva** [comment]
don't know if this is a patchtester issue or the isis flat repaint, but look at the mouseover in this buttons
image

As for the rest is looks good

**Comment by andrepereiradasilva** [comment]
also notice some gray model Close/Cancel buttons mouseover border is blue. not surre again if related to the isis flat repaint

**Comment by ciar4n** [comment]
Thanks again @andrepereiradasilva 

All template related. All buttons should now hover in the correct state with the latest commit.

**Comment by andrepereiradasilva** [comment]
have to say @ciar4n nice work! really love this flat design. much better IMO!

**Comment by andrepereiradasilva** [comment]
again not sure if related to this but look at the permissions tab
image

**Comment by ciar4n** [comment]
Thank you @andrepereiradasilva although I just put some finishing touches on the work started by @dgt41 

For some it may take a little getting use to but I think in time even they will appreciate the cleaner UI.

Well spotted. I have re-aligned the permissions dropdown in the latest commit.

permissions

**Comment by andrepereiradasilva** [comment]
> Thank you @andrepereiradasilva although I just put some finishing touches on the work started by @dgt41

ok so nice work @dgt41 too!

did you manage to check this?

> also notice some gray model Close/Cancel buttons mouseover border is blue. not surre again if related to the isis flat repaint

**Comment by ciar4n** [comment]
@andrepereiradasilva Could you detail how to replicate this issue? Unable to find an example in Chrome or FF.

**Comment by andrepereiradasilva** [comment]
image

and some have flat borders.
image

**Comment by andrepereiradasilva** [comment]
Also noticed that some buttons text on mouseover get dark blue, some don't change the color (this is the expected i think).

**Comment by ciar4n** [comment]
This is part of the modals CSS which is loaded separately from the template. Probably best for a separate PR as there is a few changes that could be made. Just waiting to see the outcome of the improved media manager.

**Comment by andrepereiradasilva** [comment]
ok then

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on bcfaf6fae15381118ac1d1d3ab21bfd89f39ec0d<br /><br />as comments above<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12349">issues.joomla.org/tracker/joomla-cms/12349</a>.</sub>

**Comment by zero-24** [comment]
@andrepereiradasilva 

>Also noticed that some buttons text on mouseover get dark blue, some don't change the text color (this is the expected i think).

Where? All places i found look good to me also on mouseover :smile: But i need to clear the cache to make sure the new template css is loaded :smile:

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12349">issues.joomla.org/tracker/joomla-cms/12349</a>.</sub>

**Comment by zero-24** [comment]
@andrepereiradasilva please double check if that works we can set this RTC :smile: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12349">issues.joomla.org/tracker/joomla-cms/12349</a>.</sub>

**Comment by andrepereiradasilva** [comment]
@zero-24 reggarding that @ciar4n responded
> This is part of the modals CSS which is loaded separately from the template. Probably best for a separate PR as there is a few changes that could be made. Just waiting to see the outcome of the improved media manager.

So i ignore that and tested this with success

**Comment by zero-24** [comment]
RTC. Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12349">issues.joomla.org/tracker/joomla-cms/12349</a>.</sub>

**Comment by mbabker** [comment]
> don't know if this is a patchtester issue or the isis flat repaint

Just for reference the only custom CSS the patch tester uses is GitHub's octicons, and only for the GitHub logo.  Otherwise everything is themed entirely by the templates (even the Hathor overrides have no custom style rules).

**Comment by zero-24** [comment]
Patch tester works great. @mbabker I had also the issue but it is gone if you clear the cached css file with this PR.

**Pull request merged**